api/base/serializers.py

```diff
@@ -596,8 +599,8 @@ def to_esi_representation(self, value, envelope='data'):
                 return '<esi:include src="{}"/>'.format(esi_url)
 
     def format_filter(self, obj):
```

Comment:
Could you do a docstring for format_filter so people know what is expected here?